fullscreen.bs

```diff
@@ -65,8 +65,15 @@ stated otherwise it is unset.
 <p>All <a for=/>documents</a> have an associated <dfn>list of pending fullscreen events</dfn>, which
 is an <a>ordered set</a> of (<a>string</a>, <a>element</a>) <a>tuples</a>. It is initially empty.
 
-<p>To <dfn>fullscreen an <var>element</var></dfn>, set <var>element</var>'s <a>fullscreen flag</a>
-and <a for="top layer">add</a> it to its <a>node document</a>'s <a>top layer</a>.
+<p>To <dfn>fullscreen an <var>element</var></dfn>:
+
+<ol>
+ <li><p>Run {hide all pop-ups until} given null, false, and false.
```

Comment:
It would be more obvious what this does if HTML defines a "hide all popovers" method that takes a document argument. It would be used both here and by `<dialog>`. Right now, you have to find "If endpoint is null" is the algorithm to understand this ensures there are no popovers left in the top layer after this.

Reply:
I made a PR for this here: https://github.com/whatwg/html/pull/8886

Reply:
Great! Can you ping me when this is ready for review, when the dfn is exported and all?

Reply:
The algorithm has been added and exported